test/object-store/sync/flx_sync.cpp

```diff
@@ -1588,6 +1588,13 @@ TEST_CASE("flx: writes work without waiting for sync", "[sync][flx][app]") {
     });
 }
 
+TEST_CASE("flx: verify PBS/FLX websocket protocol number and prefix", "[sync][flx]") {
+    REQUIRE(8 == sync::get_current_protocol_version());
```

Comment:
this will break when we bump the version again. You should keep the initial assertion.

Reply:
This is a check to verify that the current protocol version is not updated unexpectedly - I added comments around these to make sure they are updated when the protocol version is bumped.

Reply:
Yes, my point is that going forward the checks should still be valid unless we make other changes to the prefix and we should then assert on the new version.